src/main/java/taxi/services/util/ConnectionUtil.java

```diff
@@ -0,0 +1,23 @@
+package taxi.services.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import taxi.lib.Service;
+
+@Service
+public class ConnectionUtil {
+    public static Connection getCon() {
+        Properties connectProps = new Properties();
+        connectProps.put("user", "root");
+        connectProps.put("password", "12341234");
+        String url = "jdbc:mysql://localhost:3306/storage";
```

Comment:
init_db says that db is called `taxiStorage`?

Reply:
fixed